x-pack/plugins/ml/public/embeddables/anomaly_charts/anomaly_charts_react_container.tsx

```diff
@@ -58,15 +64,38 @@ const AnomalyChartsContainer: FC<AnomalyChartsContainerProps> = ({
   onLoading,
   api,
 }) => {
+  const isMounted = useMountedState();
+
+  const [tableData, setTableData] = useState<AnomaliesTableData>({
+    anomalies: [],
+    examplesByJobId: [''],
+    interval: 0,
+    jobIds: [],
+    showViewSeriesLink: false,
+  });
+
   const [chartWidth, setChartWidth] = useState<number>(0);
   const [severity, setSeverity] = useState(
     optionValueToThreshold(
       severityThreshold !== undefined ? severityThreshold : ML_ANOMALY_THRESHOLD.WARNING
     )
   );
   const [selectedEntities, setSelectedEntities] = useState<MlEntityField[] | undefined>();
-  const [{ uiSettings }, { data: dataServices, share, uiActions, charts: chartsService }] =
-    services;
+  const [
+    {
+      uiSettings,
+      notifications: { toasts },
+    },
+    { data: dataServices, share, uiActions, charts: chartsService },
+    { mlApi },
+  ] = services;
+
+  const mlJobService = useMemo(
+    () => mlJobServiceFactory(toastNotificationServiceProvider(toasts), mlApi),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
```

Comment:
Unless there are performance issues, we should move away from suppressing these when possible to avoid potentially hiding real issues.

Reply:
Now with the `rip-dependency-cache` PR merged I was able to replace this with `useMlJobService` in [5117507](https://github.com/elastic/kibana/pull/186587/commits/511750730054fea38c1a63cd5050948100b95995) so this `useMemo` is gone.